clojure-mode.el

```diff
@@ -1761,7 +1770,10 @@ If PATH is nil, use the path to the file backing the current buffer."
          (sans-file-sep (mapconcat 'identity (cdr (split-string sans-file-type "/")) "."))
          (sans-underscores (replace-regexp-in-string "_" "-" sans-file-sep)))
     ;; Drop prefix from ns for projects with structure src/{clj,cljs,cljc}
-    (replace-regexp-in-string "\\`clj[scx]?\\." "" sans-underscores)))
+    (cl-reduce (lambda (a x)
+                 (replace-regexp-in-string x "" a))
+               (add-to-list 'clojure-elided-prefixes-ns "\\`clj[scx]?\\.")
```

Comment:
Why don't you just make this the default value? (or expand it to whatever you had suggested in the message)

Reply:
Good point I'll make it the default.